app/views/declaration/supervising_customs_office.scala.html

```diff
@@ -42,50 +40,32 @@
 
 @(mode: Mode, form: Form[SupervisingCustomsOffice])(implicit request: JourneyRequest[_], messages: Messages)
 
-@pageHint = {
-    <span class="govuk-hint">
-        @messages("declaration.warehouse.supervisingCustomsOffice.hint")
-    </span>
-}
-
 @tariff = {
     @components.details(messages("site.details.summary_text_these")) {
         @Html(messages("declaration.type.supervisingCustomsOfficeSummaryText", components.details_content_link()))
     }
 }
 
-@autocomplete = {
-    @components.fields.field_accessible_autocomplete(
-        field = form("supervisingCustomsOffice"),
-        label = "",
-        labelClass = None,
-        hintText = None,
-        emptySelectValue = messages("declaration.warehouse.supervisingCustomsOffice.label"),
-        items = AutoCompleteItem.fromSupervisingCustomsOffice(CustomsOffices.all)
-    )
-}
-
 @govukLayout(
     title = Title("declaration.warehouse.supervisingCustomsOffice.title"),
     backButton = Some(BackButton(messages("site.back"), Navigator.backLink(SupervisingCustomsOffice, mode)))){
 
     @formHelper(action = controllers.declaration.routes.SupervisingCustomsOfficeController.submit(mode), 'autoComplete -> "off") {
         @errorSummary(form.errors)
 
-        @sectionHeader(messages("declaration.warehouse.supervisingCustomsOffice.sectionHeader"))
+        @components.fields.field_accessible_autocomplete_with_header(
```

Comment:
Previous implementation used fieldset. On this page AXE show two errors related with labels.

One input on page shouldn't be wrapped in fieldset. I believe that this will solve AXE error, because this error doesn't appear on rest of the pages

Reply:
Nit: I think you could remove `govukFieldset` from constructor.

Reply:
@MaciejRewera Could you take a look once again?

Sorry I put changes into the same commit accidentally. Commit is small so it shouldn't be a problem :)

Reply:
👍  I made similar changes to other "single question pages" in order to correctly display validation errors